components/slider.json

```diff
@@ -0,0 +1,15 @@
+{
+    "slider": {
```

Comment:
"slider" is not the name the design team uses in figma. To make it consistent with it, let's use 'videoControlSlider' or 'sliderButton'

Reply:
I just double checked the Figma file, it has component name "slider", but under "Control" section. This component is actually not just used in Video Control. It is also used in Audio Control, Layout Control etc. 
I assume "slider" is fine?